rover_msgs/DriveStateData.lcm

```diff
@@ -1,6 +1,9 @@
 package rover_msgs;
 
 struct DriveStateData {
-	int8_t controller; //0 = left, 1 = right
-    int8_t state; //0="Boot", 1="Disarmed", 2="Armed", 3="Calibrating", 4="Error", or 5="Exit"
+    const int8_t ODRIVE_FRONT_WHEELS = 0;
```

Comment:
where are these used?

Reply:
they're not, I guess I can just leave them as comments